be/src/runtime/stream_load/stream_load_context.h

```diff
@@ -40,26 +41,39 @@ class KafkaLoadInfo {
 public:
     KafkaLoadInfo(const TKafkaLoadInfo& t_info):
         brokers(t_info.brokers),
-        group_id(t_info.group_id),
-        client_id(t_info.client_id),
         topic(t_info.topic),
-        max_interval_s(t_info.max_interval_s),
-        max_batch_rows(t_info.max_batch_rows),
-        max_batch_bytes(t_info.max_batch_size),
         begin_offset(t_info.partition_begin_offset) {
+
+        if (t_info.__isset.max_interval_s) {
+            max_interval_s = t_info.max_interval_s;
+        }
+
```

Comment:
can you drop this empty line?

I think this is harmful to read code.

If there is no complex logic block, please don't add empty line

Reply:
I will put it to one line